.github/workflows/kubeflow-release.yaml

```diff
@@ -0,0 +1,140 @@
+---
+name: Kubeflow Release Pipeline
+on:
+  workflow_dispatch:
+    inputs:
+      create-new-release:
+        description: "Create a new release?"
+        required: true
+        default: "true"
+env:
+  CREATE_NEW_RELEASE: ${{ inputs.create-new-release }}
+  REPO_OWNER: opendatahub-io
+  REPO_NAME: kubeflow
+  BRANCH_NAME: v1.9-branch
+
+jobs:
+  # 1. Sync changes to opendatahub:v1.9-branch from opendatahub:main
+  sync-main-to-release-branch:
+    uses: opendatahub-io/kubeflow/.github/workflows/sync-branches.yaml@main
+    with:
+      source: "main"
+      target: "v1.9-branch"
+
+  # 2. Poll for images to be available on quay.io the readiness of the images usually takes ~10 mins
+  wait-images-to-build-on-quay:
+    needs: sync-main-to-release-branch
+    runs-on: ubuntu-latest
+    outputs:
+      images_ready: ${{ steps.check-images.outputs.images_ready }}
+    steps:
+      - name: Poll for images availability
+        id: check-images
+        run: |
+          # Install required tools
+          sudo apt-get update
+          sudo apt-get install -y skopeo jq curl
+
+          # Get the latest Git hash from the target branch
+          PAYLOAD=$(curl --silent -H 'Accept: application/vnd.github.v4.raw' https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/commits?sha=$BRANCH_NAME&per_page=1)
+          GIT_HASH=$(echo "$PAYLOAD" | jq -r '.[0].sha' | cut -c 1-7)
```

Comment:
I personally like using the `gh` CLI (which is available OOB on GHA) whenever possible.

Similar command using `gh api`:
```
gh api -XGET  -H "Accept: application/vnd.github+json"   -H "X-GitHub-Api-Version: 2022-11-28"  -f sha=v1.9-branch -f per_page=1 /repos/{owner}/{repo}/commits --jq '.[0].sha' | cut -c 1-7

34c2732
```

Totally fine to ignore this comment if `curl` preferred.

Reply:
Great suggestion,  I tried to do this:
`gh api -XGET  -H "Accept: application/vnd.github+json"   -H "X-GitHub-Api-Version: 2022-11-28"  -f sha=v1.9-branch -f per_page=1 /repos/$REPO_OWNER/$REPO_NAME/commits --jq '.[0].sha' | cut -c 1-7`

But i get the following error:
```
gh: Not Found (HTTP 404)
{"mess`
```

However the If i try other commands with `gh` like: `gh repo view $REPO_OWNER/$REPO_NAME` seems working
```
gh repo view $REPO_OWNER/$REPO_NAME
atheo89/kubeflow
Machine Learning Toolkit for Kubernetes


   ODH Kubeflow                                                                                                       
                                                                                                                      
  This repository is forked from the Kubeflow upstream                                                                
  repository https://github.com/kubeflow/kubeflow.                                                                    
                                                                                                                      
  ## Rebase                                                                                                           
                                                                                                                      
  Follow the instructions in the REBASE.md https://github.com/atheo89/kubeflow/blob/main/REBASE.md file to understand 
  how                                                                                                                 
  to rebase this repository without conflicts.                                                                        
                                                                                                                      
  whatever                                                                                                            
  whtaever2                                                                                                           

View this repository on GitHub: https://github.com/atheo89/kubeflow

```
I will not apply this suggestion for now but i will keep it for future enhancement, with proper time

Reply:
Interesting.. I'll play around with that next week too - sorry for bad advice there